src/services/session.ts

```diff
@@ -7,7 +7,7 @@ type Session = {
   SessionEndDate: string;
 };
 
-type DaysLeft = [daysRemaining: number, daysCompleted: number];
+type DaysLeft = [daysRemaining: number, daysInSemester: number];
```

Comment:
Has this been changed in the API? Was it named incorrectly before?

Reply:
So this is a JS "tuple", aka an array. So the names are only for TypeScript/Intellisense reference, the elements of the array aren't actually named. Also, this is the correct name, matching the one in the API. It was incorrect before but inconsequential.

Reply:
Oh yes, thanks for clarifying for me. Great!